src/cli/cli.cpp

```diff
@@ -2947,25 +2947,6 @@ otError Interpreter::ProcessNeighbor(uint8_t aArgsLength, Arg aArgs[])
 }
 #endif // OPENTHREAD_FTD
 
-#if OPENTHREAD_CONFIG_PLATFORM_NETIF_ENABLE
-otError Interpreter::ProcessNetif(uint8_t aArgsLength, Arg aArgs[])
```

Comment:
RTOS platforms may also want this command and they will need to duplicate the code for posix platforms. Thoughts?

Reply:
I don't think this is a common command. And platforms should be able to have their flexibility to writer their own.